### PR TITLE
Added solution

### PR DESCRIPTION
**Pull request by AndrewPolishchuc**

### CODE REVIEW

File: src/main/java/core/basesyntax/FileWork.java [hunk 1]
```diff
@@ -8,7 +16,29 @@
...
+    public static String[] readFromFile(String fileName) {
+        String sampleLine;
+        File sampleFile = new File(fileName);
+        ArrayList<String> answerList = new ArrayList<>();
+        try (BufferedReader sampleBuffer = new BufferedReader(new FileReader(sampleFile))) {
+            while ((sampleLine = sampleBuffer.readLine()) != null) {
+                String[] words = sampleLine.split(" ");
+                for (String word : words) {
+                    if (word.toLowerCase().charAt(0) == SEARCHLETTER) {
+                        answerList.add(word.toLowerCase().replaceAll("[^A-Za-zА-Яа-я0-9]",""));
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/FileWork.java:30]:
```suggestion
                        answerList.add(word.toLowerCase().replaceAll("\\W+",""));
```

File: src/main/java/core/basesyntax/FileWork.java [hunk 2]
```diff
@@ -8,7 +16,29 @@
...
+                    }
+                }
+            }
+        } catch (FileNotFoundException e) {
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/FileWork.java:34]: try catching both Exceptions in the same catch block using pipe `|`

File: src/main/java/core/basesyntax/FileWork.java (hunk 1 above)
Comment (AndrewPolishchuc) [reply src/main/java/core/basesyntax/FileWork.java]: Done

File: src/main/java/core/basesyntax/FileWork.java (hunk 2 above)
Comment (AndrewPolishchuc) [reply src/main/java/core/basesyntax/FileWork.java]: We can`t use | for catching both Exceptions. Types in multi-catch must be disjoint: 'java.io.FileNotFoundException' is a subclass of 'java.io.IOException'. We can only catch java.io.IOException.

**Pull request closed without merging**